src/utils/node.ts

```diff
@@ -67,11 +76,96 @@ const copyLegacyDB = async (_this: Command) => {
 	_this.log(`Legacy database for Lisk Core v4 has been created at ${LEGACY_DB_PATH}`);
 };
 
-const getFinalConfigPath = async (outputDir: string, network: string) =>
+export const getFinalConfigPath = async (outputDir: string, network: string) =>
 	(await exists(`${outputDir}/config.json`))
 		? outputDir
 		: path.resolve(__dirname, '../..', 'config', network);
 
+export const validateStartCommandParams = async (
+	allowedFlags: string[],
+	userInputString: string,
+): Promise<boolean> => {
+	try {
+		let isOptionOrValueExpected = false;
+
+		const userInputs = userInputString.split(REGEX.INPUT_SEPARATOR);
+		for (let i = 0; i < userInputs.length; i += 1) {
+			const input = userInputs[i];
+
+			// Since network is determined automatically, user shouldn't pass the network flag
+			if (REGEX.NETWORK_FLAG.test(input)) return false;
+
+			if (REGEX.FLAG.test(input)) {
+				const correspondingFlag = allowedFlags.find(f => f.includes(input));
+
+				// If unknown flag specified or expected a value for prev flag or if no value provided when expected
+				if (!correspondingFlag || isOptionOrValueExpected || i === userInputs.length - 1) {
+					return false;
+				}
+
+				if (REGEX.OPTION_OR_VALUE.test(correspondingFlag)) {
+					isOptionOrValueExpected = true;
+				}
+			} else if (isOptionOrValueExpected) {
+				isOptionOrValueExpected = false;
+			} else {
+				return false;
+			}
+		}
+
+		return true;
+	} catch (error) {
+		return false;
+	}
+};
+
+const resolveLiskCoreStartCommand = async (_this: Command, network: string, configPath: string) => {
+	const isUserConfirmed = await cli.confirm(
+		'Would you like to customize the Lisk Core v4 start command params? [yes/no]',
+	);
+
+	if (!isUserConfirmed) {
+		const defaultStartCommand = `lisk core start --network ${network} --config ${configPath}/config.json`;
+		return defaultStartCommand;
+	}
+
+	// Let user customize the start command
+	let customStartCommand;
+
+	_this.log('Customizing Lisk Core start parameters');
+	let userInput = await cli.prompt(
+		"Please provide the Lisk Core start command params you would like to specify except the '--network (-n)' flag (e.g. --api-ws):",
+	);
+
+	const command = "lisk-core start --help | grep -- '^\\s\\+-' | cut -d ' ' -f 3,4";
+	const allowedFlags = await execAsync(command);
+	const allowedFlagsArray = allowedFlags.split(/\n+/).filter(e => !!e);
+
+	let numTriesLeft = 3;
+	while (numTriesLeft) {
+		numTriesLeft -= 1;
+
+		const isValidParams = await validateStartCommandParams(allowedFlagsArray, userInput);
+		if (isValidParams) {
+			/* eslint-disable-next-line @typescript-eslint/restrict-template-expressions */
+			customStartCommand = `lisk core start --network ${network} ${userInput}`;
+			break;
+		}
+
+		if (numTriesLeft >= 0) {
+			userInput = await cli.prompt(
+				"Invalid params passed, please provide the Lisk Core start command params you would like to specify except the '--network (-n)' flag (e.g. --api-ws):",
```

Comment:
params -> flags? 

cross check other places if we are using flags and params correctly.. also in CLI its used like

command -> args e.g `lisk-core transaction:create token transfer 100000000` where module command and fees are args
command -> flags e.g `lisk-core start --network=testnet` with -- are flags